lineman/app/services/thread_query_service.coffee

```diff
@@ -0,0 +1,64 @@
+angular.module('loomioApp').factory 'ThreadQueryService', (Records) ->
+  new class ThreadQueryService
+
+    unreadQuery: ->
+      threadQueryFor createBaseView(), 'show_unread'
+
+    groupQuery: (group = {}, options = {}) ->
+      threadQueryFor createGroupView(group), 
+                     options['filter'] or 'show_unread',
+                     options['limit'] or 5
+
+    timeframeQuery: (options = {}) ->
+      threadQueryFor createTimeframeView(options['name'], options['timeframe']),
+                     options['filter'] or 'show_all',
+                     options['limit'] or 100
+
+    threadQueryFor = (view, filter) ->
+      view: view
```

Comment:
Here's our query object... which will likely make its way to being a full fledged model at some point.

It's basically a db view (which stays the same) + a filter (which can change) + helper methods for data access goodness.

I've had a great time with this setup so far.